src/components/Reports/SubscriberHistory/SubscriberHistorySubscriber/SubscriberHistorySubscriber.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import { InjectAppServices } from '../../../../services/pure-di';
+import { Loading } from '../../../Loading/Loading';
+import { FormattedMessage } from 'react-intl';
+
+const SubscriberHistorySubscriber = ({ email, dependencies: { dopplerApiClient } }) => {
```

Comment:
Why not rename this into something like `CurrentSubscriberResult`? something indicating this component deals with the search of a subscriber inside subscriberHistory component. `SearchedSubscriber` `SubscirberResult` also, you know you are within subscribersHistory because of the folder structure.

Reply:
I've talked with @fcoronelmakingsense and he explained that we use parent component as part of the name, because the folders are for us humans only. Then it all get scrambled O_o I've suggested then `SubscriberHistoryCurrentSubscriber` or `SubscriberHistorySearchedSubscriber`